src/opm/output/eclipse/EclipseIO.cpp

```diff
@@ -576,14 +570,9 @@ RestartValue EclipseIO::loadRestart(const std::vector<RestartKey>& solution_keys
                                                                         report_step,
                                                                         false );
 
-    auto rst = RestartIO::load(filename, report_step, solution_keys,
+    auto rst = RestartIO::load(filename, report_step, summary_state, solution_keys,
                                es, grid, schedule, extra_keys);
-
-    // Technically a violation of 'const'.  Allowed because 'impl' is
-    // constant pointer to mutable Impl.
-    this->impl->summary.reset_cumulative_quantities(rst.second);
-
-    return std::move(rst.first);
+    return std::move(rst);
```

Comment:
Since `RestartIO::load()` no longer returns a `pair`, this can just be
```C++
return RestartIO::load(/* ... */);
```
There's no need to capture the return value in a local variable and then to `move()` it out.

Reply:
OK - fixed